concepts/05 UI Components/DataGrid/75 Keyboard Support.md

```diff
@@ -72,7 +73,7 @@ A user can interact with the UI component using the following keys:
     </tr>
     <tr>
         <td>Tab / Shift + Tab</td>
-        <td>Moves the focus one cell forward/back.<br /> Focuses the first/last cell in the next/previous row if cells in the current row run out.</td>
+        <td>Moves the focus one element forward/back.<br /> Focuses the first/last element in the next/previous row if elements in the current row run out.</td>
```

Comment:
"in the current row run out" Explain

Reply:
![image](https://user-images.githubusercontent.com/66619750/105326197-8a887c80-5bde-11eb-8cb5-2466d82c3762.png)
The next focused element is 'Olivia'.